src/components/Table/Table.stories.tsx

```diff
@@ -158,3 +158,26 @@ export const FixedWidth: Story = {
     ]
   }
 };
+
+const maxUidTestRows = [...Array.from({ length: 100 }).keys()].map(
+  (item, index) => [
+    index + 10_000,
```

Comment:
Consider making the Column 1 values something that would demonstrate the difference between `cellDisableWordWrap: true` and `cellDisableWordWrap: false`.  When only numbers, the current row labels don't change. 

```suggestion
    `Row ${index + 1_000_000_000}`,
```